### DISCUSSION TITLE
Line error and hint not shown

### DISCUSSION THREAD

**Discussion by mcaballeropinto** [mtxr/vscode-sqltools#1285] [Q&A]
Hi there!

I love this extension and I subscribe to the idea of querying my DBs right on my IDE. However, there is a major gap vs other tools that is keeping me from using it. 

Let me explain it with an example, when there is a syntax error in my query and I run the query on Postico I get a message like this:

```
ERROR:  invalid reference to FROM-clause entry for table "accounts"
LINE 80:       user_workspaces ->> 'account_id' = accounts.id::text
                                                  ^
HINT:  There is an entry for table "accounts", but it cannot be referenced from this part of the query.
```

Notice how the error shows the exact line and character where it happened and it shows a hint too.

When I run that exact query in VS SQLTools extension, I get this:

```
invalid reference to FROM-clause entry for table "accounts"
```

Notice how the line number is missing.

I in this case I am querying a Postgres DB and have the SQLTools PostgreSQL/Cockroach Driver extension for the connector.

Is there a way to show the line error and the hint on this extension?

Thanks.

**Comment by gjsjohnmurray** [comment]
There is an open issue for this at https://github.com/mtxr/vscode-sqltools/issues/846
